src/impl/formatter.js

```diff
@@ -375,29 +375,30 @@ export default class Formatter {
     const tokenToField = (token) => {
         switch (token[0]) {
           case "S":
-            return "millisecond";
+            return "milliseconds";
           case "s":
-            return "second";
+            return "seconds";
           case "m":
-            return "minute";
+            return "minutes";
           case "h":
-            return "hour";
+            return "hours";
           case "d":
-            return "day";
+            return "days";
           case "w":
-            return "week";
+            return "weeks";
           case "M":
-            return "month";
+            return "months";
           case "y":
-            return "year";
+            return "years";
```

Comment:
Why were these changed?

Reply:
So that `mapped` would match the `Object.keys` of `lildur.values`, and because `lildur.get` doesn't care whether it is plural or not.